11ty/definitions/spiritual-bypassing.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Spiritual Bypassing
+slug: spiritual-bypassing
+defined: true
+excerpt: a form of appropriation where non-Black individuals use memes and imagery featuring Black people and dark-skinned emojis
```

Comment:
```suggestion
excerpt: a form of appropriation where non-Black individuals use memes and imagery featuring Black people and dark-skinned emojis
```
I think this needs updating

Reply:
Oh whoops, sorry about that. Guess which term I duplicated to get started? 😬